app/javascript/angular/tests/_sensors.test.js

```diff
@@ -211,7 +22,7 @@ test('selected with no sensor id in the url returns the default sensor with adde
 
   const expected = {
     ...defaultSensor,
-    id: 'Particulate Matter-airbeam2-pm2.5 (µg/m³)',
+    id: 'particulate matter-airbeam2-pm2.5 (µg/m³)',
```

Comment:
Pina decided on that with Micheal. But it's a good idea to confirm during the call, I'll do that.
The side effect is that old links with parameters capitalized will not work anymore, but since we are changing bigger things in the API, I guess we can accept that.